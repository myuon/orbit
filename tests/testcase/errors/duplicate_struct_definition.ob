type Person = struct {
  name: string,
  age: int,
};

type Person = struct {
  id: int,
  email: string,
};

fun main() do
  return 42;
end