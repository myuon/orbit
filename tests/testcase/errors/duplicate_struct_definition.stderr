Struct 'Person' is already defined